iree/compiler/Codegen/SPIRV/Utils.cpp

```diff
@@ -22,6 +22,8 @@
 namespace mlir {
 namespace iree_compiler {
 
+static constexpr int kNumGPUDims = 3;
```

Comment:
Is this used anywhere?

Reply:
Yes used later. Now changed to getting the definition from GPUUtils.h.